### PR TITLE
Security Guide

### PR DESCRIPTION
**Pull request by lmalivert**
Security guide - Firewall Policy Management ready for review.

### CODE REVIEW

File: downstream/assemblies/security/assembly-firewall-policy-manage.adoc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+include::security/con-about-firewall-policy-management.adoc[leveloffset=+1]
+include::security/con-automating-firewall-rules.adoc[leveloffset=+1]
+
+////
+[leveloffset=+1] ensures that when a module title is a level 1 heading (= Title), the heading will be interpreted as a level-2 heading (== Title) in the assembly. Use [leveloffset=+2] and [leveloffset=+3] to nest modules in an assembly.
+////
+
+include::security/proc-creating-firewall-rule.adoc[leveloffset=+2]
+include::security/proc-deleting-firewall-rule.adoc[leveloffset=+2]
+
```
Comment (samccann) [inline downstream/assemblies/security/assembly-firewall-policy-manage.adoc:38]: There's no real reason to make these procedures a subsection of the automat firewall rules concept module. Consider changing  leveloffset=+1 for both.

Comment (lmalivert) [reply downstream/assemblies/security/assembly-firewall-policy-manage.adoc:38]: @samccann Thanks for the review. I changed it to leveloffset=+1.

**Review by samccann — APPROVED** [review]
Some minor things we can fix post release but going to approve and merge as is for now so we make the release.

**Comment by lmalivert** [comment]
@samccann Wondering, what minor things are you referring to?

**Comment by samccann** [comment]
There's some asciidoc formatting we should put around a few things. The playbook output in the screencaps isn't an ansible-navigator output, but it's close enough that nobody but me would probably notice, etc. We'll either find a way to rerun  them ourselves or ask your security buddies to rerun it with the final release of navigator, etc.

**Comment by lmalivert** [comment]
@samccann Oh okay, got it.

**Pull request merged**